docs/source/migration.mdx

```diff
@@ -339,7 +339,7 @@ If you've written your own extensions (passed to `new ApolloServer({extensions:
 
 In later versions of Apollo Server (including Apollo Server 3), you instead provide this configuration via a combination of the `apollo` constructor option, plugins, and `APOLLO_`-prefixed environment variables. Apollo Server 3 does not support the `engine` constructor option or the `ENGINE_`-prefixed environment variables.
 
-If your project still uses the `engine` option or the `ENGINE_`-prefixed environment variables, see [Migrating from the `engine` option](./v2/migration-engine-plugins) before upgrading to Apollo Server 3.
+If your project still uses the `engine` option or the `ENGINE_`-prefixed environment variables, see [Migrating from the `engine` option](https://www.apollographql.com/docs/apollo-server/v2/migration-engine-plugins) before upgrading to Apollo Server 3.
```

Comment:
I _think_ this can just be `/apollo-server/v2...` with the new docs architecture